gui/default/syncthing/core/aboutModalView.html

```diff
@@ -2,17 +2,21 @@
   <div class="modal-body">
     <h1 class="text-center">
       <img alt="Syncthing" src="assets/img/logo-horizontal.svg" style="max-width: 366px; vertical-align: -16px" />
-      <br />
+    </h1>
+    <h1 ng-if="version.version" class="text-center">
```

Comment:
What's the reason for the changes to this file? This page is not available when not authenticated, so I'd expect that no change is needed here at all.

Reply:
It is now. :slightly_smiling_face: But parts of it aren't available yet, that's what the changes are for.

Reply:
Oh I see, I missed that you did not disable the entry for the about page in the actions menu - I just assumed that would be disabled, but I guess most of the info is present and valid even if not authenticated.